packages/terra-pills/src/private/_PillsUtils.js

```diff
@@ -0,0 +1,49 @@
+const getPillWidth = (pill) => {
```

Comment:
All the functions in this file need description comments

Reply:
Personally I feel functions like these are self-explanatory. If I had to write a docstring I'm not sure what value it would have beyond re-stating the name and input varialbe name - which itself is explaining the type (a pill). Same goes for the rest of the functions in this particular module. 

Unless the docstring provides some benefit, why include it? Does it get scraped up to produce site doc or something like that?

Reply:
added 40ee383faaf027a3fbbd63aec56cf8e88f282029